app/controllers/api/v1/categories_controller.rb

```diff
@@ -3,6 +3,25 @@
 module Api
   module V1
     class CategoriesController < ApplicationController
+      # POST /categories
+      def create
+        @category = Category.new(category_params)
```

Comment:
Acá también podes usar el exception handler que te comenté en el otro PR y usar directamente:
```suggestion
        @category = Category.create!(category_params)
```